Snowfakery Version 2.4 (#573)
SFDO-Tooling/Snowfakery: HISTORY.md

And so [Salesforce.org](http://salesforce.org/) said “Let there be data,” and there was Snowfakery. And it was good.

## Snowfakery 2.4

> The Salesforce load order now takes into account the order in which

Reviewer: ```suggestion
The Salesforce load order now follows the order of
```

> object templates were declared in a recipe. The other considerations

Reviewer: ```suggestion
object template declarations in the recipe. The other considerations
```

Reviewer: I believe there's one recipe when you're running Snowfakery and that it now determines the load order. If I'm wrong, my suggestions are likely wrong too. Let me know.
